src/base/http/responsebuilder.h

```diff
@@ -30,14 +30,15 @@
 #define HTTP_RESPONSEBUILDER_H
 
 #include <QObject>
+
```

Comment:
The same thing: no need to split these includes since there are only one include in each group. It's readable as is.